packages/core/theme-module/theme/components/LocaleSelector.vue

```diff
@@ -8,16 +8,12 @@
     >
       <SfImage :src="`/icons/langs/${lang.name}.png`" width="20" />
     </SfButton>
-    <SfSelect v-model="country" class="container__select">
-      <SfSelectOption v-for="currentCountry in availableCountries" :key="currentCountry.name" :value="currentCountry.name">
-        <div>{{ currentCountry.label }}</div>
-      </SfSelectOption>
-    </SfSelect>
-    <SfSelect v-model="currency" class="container__select">
+    <!-- TODO: replace with SfDropdown or dedicated component -->
```

Comment:
Can you replace it by yourself?

Reply:
Yes, this will be done as soon as SFUI theme will finish the LanguageSelector component